### DISCUSSION TITLE
Multiple repeat pattern

### DISCUSSION THREAD

**Discussion by bombnattanon** [Kinggerm/GetOrganelle#307] [Q&A]
Hi,

Thank you for developing this amazing software! It works with assembling plastomes of 95% of my samples across 70 land plant families!

I have come across several samples where GetOrganelle produced multiple (hundreds or a thousand) of results with different repeat patterns. 

I attached an example of the assembly graph below. This plastome failed assembled when using the default word size (was estimated as 115) with k 35, 55, 85, 115, 127; and successfully circularised when I changed the word size to 65 with the same k setting. However, it produced 1000 outputs (I guess that's the limit) with different repeat patterns. I was wondering whether there's any other parameters I should readjust to get a better result?

Thank you very much!

![graph](https://github.com/Kinggerm/GetOrganelle/assets/56253006/db78299d-bf49-4765-8514-465a1a5aa2a2)

<img width="556" alt="image" src="https://github.com/Kinggerm/GetOrganelle/assets/56253006/4a5e0226-743f-4be8-bdc5-d7965c0b6391">

**Comment by JianjunJin** [comment]
Thank you for your detailed query and for using GetOrganelle. 

In cases with reads shorter than the repeats, our options to enhance results are indeed limited. However, if your paired-end library size is relatively long (e.g., around 500 bp, exceeding the repeat lengths), you might find some resolution in the scaffolding information provided by SPAdes for certain tangled cases. I'll consider integrating this solution in future versions of GetOrganelle if it proves effective.
